server/game/basecard.ts

```diff
@@ -364,8 +364,8 @@ class BaseCard extends EffectSource {
         }
     }
 
-    canTriggerAbilities(context: AbilityContext): boolean {
-        return !this.facedown && this.checkRestrictions('triggerAbilities', context);
+    canTriggerAbilities(context: AbilityContext, ignoredRequirements = []): boolean {
+        return !this.facedown && (ignoredRequirements.includes('triggeringRestrictions') || this.checkRestrictions('triggerAbilities', context));
```

Comment:
Looking at the merge that needs to happen `|| this.checkRestrictions('triggerAbilities', context))` actually is a && case. Any reason why this changed into OR here?

Reply:
My best guess is that I changed it from
!facedown AND [no restrictions on triggering abilities]
to
!facedown AND [either (I ignoring triggering restrictions) or (there are no triggering restrictions)]

Reply:
Ah yes, looked at it more closely.  We used the same pathways to do "Trigger the ability" vs "Resolve the ability" (ex: Cunning Negotiator vs Countryside Trader).  So I had to put a "ignore triggering restriction" when we do direct "resolve the ability".